drivemount/drive-list.c

```diff
@@ -39,33 +39,34 @@ static void drive_list_dispose  (GObject *object);
 static void drive_list_add      (GtkContainer *container, GtkWidget *child);
 static void drive_list_remove   (GtkContainer *container, GtkWidget *child);
 
-static void mount_added        (GVolumeMonitor *monitor,
-				GMount *mount,
-				DriveList *self);
-static void mount_changed      (GVolumeMonitor *monitor,
-				GMount *mount,
-				DriveList *self);
-static void mount_removed      (GVolumeMonitor *monitor,
-				GMount *mount,
-				DriveList *self);
-static void volume_added       (GVolumeMonitor *monitor,
-				GVolume *volume,
-				DriveList *self);
-static void volume_changed     (GVolumeMonitor *monitor,
-				GVolume *volume,
-				DriveList *self);
-static void volume_removed     (GVolumeMonitor *monitor,
-				GVolume *volume,
-				DriveList *self);
-static void add_volume         (DriveList *self,
-				GVolume *volume);
-static void remove_volume      (DriveList *self,
-				GVolume *volume);
-static void add_mount          (DriveList *self,
-				GMount *mount);
-static void remove_mount       (DriveList *self,
-				GMount *mount);
-static void queue_relayout     (DriveList *self);
+static void mount_added         (GVolumeMonitor *monitor,
+                                 GMount *mount,
+                                 DriveList *self);
+static void mount_changed       (GVolumeMonitor *monitor,
+                                 GMount *mount,
+                                 DriveList *self);
+static void mount_removed       (GVolumeMonitor *monitor,
+                                 GMount *mount,
+                                 DriveList *self);
+static void volume_added        (GVolumeMonitor *monitor,
+                                 GVolume *volume,
+                                 DriveList *self);
+static void volume_changed      (GVolumeMonitor *monitor,
+                                 GVolume *volume,
+                                 DriveList *self);
+static void volume_removed      (GVolumeMonitor *monitor,
+                                 GVolume *volume,
+                                 DriveList *self);
+static void add_volume          (DriveList *self,
+                                 GVolume *volume);
+static void remove_volume       (DriveList *self,
+                                 GVolume *volume);
+static void add_mount           (DriveList *self,
+                                 GMount *mount);
+static void remove_mount        (DriveList *self,
+                                 GMount *mount);
+static void queue_relayout      (DriveList *self);
+
```

Comment:
You don't keep the criteria with the functions that have the drive_list prefix. IMHO, the signatures can be defined using a single line in this case.

Reply:
That wasn't me, i only fixed the alignment :) 
But, np, i can rewrite it in a single line.